src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -159,6 +159,10 @@ protected override void Apply(TransactionOperationContext context, BlittableJson
                     case nameof(DeleteOngoingTaskCommand):
                     case nameof(AddRavenConnectionString):
                     case nameof(AddSqlConnectionString):
+                    case nameof(RemoveRavenConnectionString):
+                    case nameof(RemoveSqlConnectionString):
+                    case nameof(UpdateRavenConnectionString):
```

Comment:
Do we really need an Add an Update commands? Can't we do with just Put that will update if it is already there?